tests/kafkatest/services/verifiable_producer.py

```diff
@@ -63,6 +65,11 @@ def start_cmd(self):
             cmd += " --max-messages %s" % str(self.max_messages)
         if self.throughput > 0:
             cmd += " --throughput %s" % str(self.throughput)
+        cmd += " --close-timeout %s" % str(self.close_timeout_sec)
+
+        if self.producer_props is not None:
```

Comment:
Hm. I suppose it's ok to have a mutable default argument in the __init__ method since it will only be called once in the object's lifetime.

Changed to empty dict.